src/models/deployers/PresaleTemplateDeployer.js

```diff
@@ -0,0 +1,279 @@
+const { APP_IDS } = require('@aragon/templates-externally-owned-presale-bonding-curve/lib/helpers/constants.js')
+const BaseDeployer = require('./BaseDeployer')
+const logger = require('../../helpers/logger')('PresaleTemplateDeployer')
+const { getInstalledApps, getInstalledAppsById } = require('../../helpers/events')
+// TODO: import from https://github.com/sohkai/agent-voting or alike
```

Comment:
let's remove this TODO and reuse the `Encoder`

Reply:
Well, the TODO still applies. This helpers should be made available in some generic repo.